app/src/main/java/com/nextcloud/talk/activities/CallActivity.kt

```diff
@@ -663,7 +675,38 @@ class CallActivity : CallBaseActivity() {
                 ).show()
             }
         }
-        binding!!.hangupButton.setOnClickListener { hangup(true) }
+
+        if (isGroupConversation) {
+            binding!!.hangupButton.setOnClickListener {
+                hangup(true, null)
+            }
+        }
+
+        if (isOneToOneConversation && isModerator) {
+            binding!!.hangupButton.setOnClickListener {
+                showPopupMenu()
+            }
+        }
+
+        if (isOneToOneConversation) {
+            binding!!.hangupButton.setOnLongClickListener {
+                hangup(true, null)
```

Comment:
here, show the popup button instead.

Reply:
I clarified with @nickvergessen and @Ivansss it should also show the popup button like it is implemented in iOS (it was not in the issue description but it has now been decided that this makes more sense).
So in the end the popup button must be able to handle different actions like i mentioned in it's clickListener.